OpenFilesScreen.c

```diff
@@ -253,7 +253,7 @@ static void OpenFilesScreen_scan(InfoScreen* this) {
       while (fdata) {
          OpenFiles_Data* data = &fdata->data;
          char* entry = NULL;
-         xAsprintf(&entry, "%5.5s %-7.7s %-4.4s %-10.10s %10.10s %10.10s %10.10s  %s",
+         xAsprintf(&entry, "%5.5s %-7.7s %-4.4s %-4.4s %16.16s %10.10s %10.10s  %s",
```

Comment:
To prevent further truncation of the "SIZE" field, the precision specifier should be removed from the format. Like this:
```suggestion
         xAsprintf(&entry, "%5.5s %-7.7s %-4.4s %-4.4s %16s %10.10s %10.10s  %s",
```

Reply:
This runs it out of alignment for >10PB but that's fine I think. Applied.